src/features/donnees-personnelles/pages/donnees-personnelles/donnees-personnelles.page.html

```diff
@@ -0,0 +1,137 @@
+            </a>
+          </li>
+          <li>
+            <a href="https://www.legifrance.gouv.fr/jorf/id/JORFTEXT000044228912" target="_blank">
+              les articles 1 et 3 du décret n°2021-1362 du 20 octobre 2021.
+            </a>
+          </li>
+        </ul>
+        <h3 class="my-5 text-primary">Durée de conservation</h3>
+        <p>Les données à caractère personnel sont conservées :</p>
+        <table class="table">
+          <thead>
+            <tr>
+              <th scope="col">Types de données</th>
+              <th scope="col">Durées de conservation</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td scope="row">Données de localisation</td>
+              <td>Les données sont supprimées à compter de la prise de rendez-vous.</td>
+            </tr>
+            <tr>
+              <td scope="row">Données d'hébergeur</td>
+              <td>1 an.</td>
+            </tr>
+            <tr>
+              <td scope="row">Cookies</td>
+              <td>13 mois.</td>
+            </tr>
+          </tbody>
+        </table>
+        <h3 class="my-5 text-primary">Utilisation de témoins de connexion (« cookies »)</h3>
+        <p>
+          Les cookies utilisés sur Cartographie nationale ne sont que des cookies strictement nécessaires au bon fonctionnement
+          du site, au sens des dernières recommandations de la CNIL à ce sujet.
+        </p>
+        <p>
+          Cartographie nationale utilise également des cookies de statistiques anonymes qui ne nécessitent pas de bandeau
+          cookies, toujours au sens des dernières recommandations de la CNIL à ce sujet.
+        </p>
+        <p>Cartographie nationale n'utilise aucun cookies dits "tiers".</p>
+        <p>Il convient d'indiquer que :</p>
+        <ul>
+          <li>Les données collectées ne sont pas recoupées avec d’autres traitements.</li>
+          <li>Les cookies ne permettent pas de suivre la navigation de l’internaute sur d’autres sites.</li>
+        </ul>
+        <p>
+          À tout moment, vous pouvez refuser l’utilisation des cookies et désactiver le dépôt sur votre ordinateur en utilisant
+          la fonction dédiée de votre navigateur (fonction disponible notamment sur Microsoft Internet Explorer 11, Google
+          Chrome, Mozilla Firefox, Apple Safari et Opera).
+        </p>
+        <p>
+          Pour aller plus loin, vous pouvez consulter les fiches proposées par la Commission Nationale de l'Informatique et des
+          Libertés (CNIL) :
+        </p>
+        <ul>
+          <li>
+            <a href="https://www.cnil.fr/fr/cookies-et-autres-traceurs/regles/cookies/que-dit-la-loi" target="_blank">
+              Cookies & traceurs : que dit la loi ?
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://www.cnil.fr/fr/cookies-et-autres-traceurs/comment-se-proteger/maitriser-votre-navigateur"
+              target="_blank">
```

Comment:
```suggestion
              href="https://www.cnil.fr/fr/cookies-et-autres-traceurs/comment-se-proteger/maitriser-votre-navigateur"
              target="_blank" rel="noopener noreferrer">
```
https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener